R/read_asc_Andor.R

```diff
@@ -49,11 +49,33 @@ read_asc_Andor <- function(file = stop("filename or connection needed"),
 
 hySpc.testthat::test(read_asc_Andor) <- function() {
   context("read_asc_Andor")
-  test_that("Andor Solis .asc text files", {
-    skip("Need to adapt to new package")
-    expect_known_hash(
-      read_asc_Andor("fileio/asc.Andor/ASCII-Andor-Solis.asc"),
-      "9ead937f51"
-    )
+
+  andor <- system.file("extdata/fileio/asc.Andor/", "ASCII-Andor-Solis.asc",package = "hySpc.read.txt")
+  spc <- read_asc_Andor(andor)
+
+  test_that("Andor Solis .asc labels are correct",{
+    expect_equal(spc@label$.wavelength, NULL)
+    expect_equal(spc@label$spc, NULL)
+    expect_equal(spc@label$filename, "filename")
+  })
+
+  test_that("Andor Solis .asc spectral data", {
+    expect_equal(dim(spc@data$spc), c(5,63))
+
+    expect_equal(colnames(spc@data$spc),
```

Comment:
I asked Sang to check if wavelengths are read correctly. Just I'm wondering if we should check all the values or just a few (smallest, largest, and several intermediate). What do you think @bryanhanson, @cbeleites?

This concerns this one and especially the future unit tests.

Reply:
Checking the vector isn't any more work (using `dput()` on manually checked values)

Reply:
But I think that it would make unit tests less readable if the output of `dput()` would be added. Especially for large files.